packages/classes/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# Changelog for @tvkitchen/base-classes
```

Comment:
Could / should we use a template inspired by [this one](https://github.com/benmosher/eslint-plugin-import/blob/master/CHANGELOG.md) for our changelogs?

Basically, have the common header text that links to semver and that keepachangelog site (which we should check out!)

Reply:
Yup, that's great, and we should! I'm going to modify these with the Unreleased header, and add the actual work done thus far in each.

And [Keepachangelog](https://keepachangelog.com/en/1.0.0/) is fantastic!